src/lib/Components/Buttons/__tests__/InvertedButton-tests.tsx

```diff
@@ -3,9 +3,18 @@ import "react-native"
 import React from "react"
 import * as renderer from "react-test-renderer"
 
-import InvertedButton from "../InvertedButton"
+import { InvertedButton } from "../"
 
 it("renders properly", () => {
   const button = renderer.create(<InvertedButton text={"I am an inverted button"} />).toJSON()
   expect(button).toMatchSnapshot()
 })
+
+it("accepts textStyle prop", () => {
+  const textStyle = {
+    fontSize: 20,
+  }
+
+  const button = renderer.create(<InvertedButton text={"I am an inverted button"} textStyle={textStyle} />).toJSON()
+  expect(button).toMatchSnapshot()
+})
```

Comment:
I wasn't sure if I should add a new test so added this just to be safe, but I'm also totally fine with dropping it since this doesn't seem to be a very helpful test...